cfme/tests/containers/test_pods_prop_data_integrity.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9934
+
+
+def test_summary_properties_validation(provider):
+    """
+          This test verifies that the number of running containers in the status summary table
+          is the same number that appears in the Relationships table
```

Comment:
So, what this test does is it **goes through all Pods and checks that all containers of that pod are up and running**. I wonder why does the name of the test nor the description not reflect that fact. You are talking about data validation / integrity but if one of the container changes it's state to `Terminated` or `Waiting`, it will stop showing up under `Running` and it will break your test but there will be no issue with data integrity... However, I'm not here to judge code logic. I'm more concerned about code quality. That said, if you want this merged, it's good to merged. Just squash it first, please.